src/pages/1jeune1permis/index.page.test.tsx

```diff
@@ -123,16 +123,17 @@ describe('1jeune1permis', () => {
 
 		// When
 		const iframe = screen.getByTitle('Informations sur le dispositif 1 jeune 1 permis');
+		const data = JSON.stringify({
+			height: 2000,
+			type: 'resize-iframe',
+		});
 		fireEvent(window, new MessageEvent('message',
-			{ data: {
-				height: 2000,
-				type: 'resize-iframe',
-			},
-			origin: DOMAINE_1JEUNE_1PERMIS }),
+			{ data,
+				origin: DOMAINE_1JEUNE_1PERMIS }),
 		);
 
 		// Then
-		expect(iframe).toHaveAttribute('height', '2000');
+		expect(iframe).toHaveAttribute('style', '--1jeune1permis-iframe-height: 2000px;');
```

Comment:
Tu peux aussi faire avec `toHaveStyle()`, mais tu garde l'indirection de la _custom property_ malheureusement : 
```suggestion
		expect(iframe).toHaveStyle({
			'--1jeune1permis-iframe-height': '2000px',
		});
```
voire
```suggestion
		expect(iframe).toHaveStyle({
			'--1jeune1permis-iframe-height': '2000px',
			height: 'var(--1jeune1permis-iframe-height)',
		});
```
Je ne sais pas si c'est mieux, je te laisse décider 🤔 

**Edit**: Je n'ai pas trouvé de version équivalente au `not.toHaveAttribute('style')` avec `toHaveStyle()` qui ne rendait pas le code incompréhensible. Aucun des suivants ne fonctionne correctement : 
```js
expect(iframe).toHaveStyle(expect.not.objectContaining('--1jeune1permis-iframe-height'));
expect(iframe).not.toHaveStyle(expect.objectContaining('--1jeune1permis-iframe-height'));
expect(iframe).toHaveStyle(expect.not.stringContaining('--1jeune1permis-iframe-height'));
expect(iframe).not.toHaveStyle(expect.stringContaining('--1jeune1permis-iframe-height'));
expect(iframe).not.toHaveStyle({ '--1jeune1permis-iframe-height': expect.anything() });
```
Ça semble dût au fait que `toHaveStyle()` doit convertir en string et que par conséquent, soit il fail pour les 4 premiers, soit il check contre la string "--1jeune1permis-iframe-height: Anything" (qui du coup est toujours vrai) pour le dernier